catalog-entities/marketplace/packages/red-hat-developer-hub-backstage-plugin-global-header.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
 spec:
   packageName: "@red-hat-developer-hub/backstage-plugin-global-header"
   dynamicArtifact: ./dynamic-plugins/dist/red-hat-developer-hub-backstage-plugin-global-header
-  version: 1.5.0
+  version: 1.5.1
```

Comment:
should this be 1.5.2?

https://www.npmjs.com/package/@red-hat-developer-hub/backstage-plugin-global-header/v/1.5.2